app/src/main/res/xml/app_preferences.xml

```diff
@@ -63,11 +63,27 @@
             app:title="@string/p_battle_config_cmd_text"
             app:summary="@string/p_battle_config_cmd_text_summary" />
 
-        <SwitchPreferenceCompat
-            app:icon="@drawable/ic_time"
-            app:key="@string/pref_wait_ap_regen"
-            app:title="@string/p_wait_ap_regen_text"
-            app:summary="@string/p_wait_ap_regen_text_summary" />
+        <PreferenceCategory
+            app:key="wait_ap_regen_category"
+            app:title="@string/p_wait_ap_regen_text">
+            <SwitchPreferenceCompat
+                app:icon="@drawable/ic_time"
+                app:key="@string/pref_wait_ap_regen"
+                app:title="@string/p_wait_ap_regen_text"
+                app:summary="@string/p_wait_ap_regen_text_summary" />
+
+            <SeekBarPreference
+                android:max="60"
+                app:icon="@drawable/ic_counter"
+                app:key="@string/pref_wait_ap_regen_minutes"
+                app:title="@string/p_wait_ap_regen_minutes_text"
+                app:summary="@string/p_wait_ap_regen_minutes_text_summary"
+                app:min="1"
+                app:defaultValue="5"
```

Comment:
Good QoL overall, but can we make the default 1 min. People usually have around 1-2 min screen timeout. So, I think that should be a better default.

Reply:
Sure! let me change this and the translation too